Content.Client/ContextMenu/UI/ContextMenuElement.cs

```diff
@@ -229,15 +231,18 @@ protected override Vector2 MeasureOverride(Vector2 availableSize)
                 return Vector2.Zero;
             }
 
-            List.Measure(availableSize);
-            var listSize = List.DesiredSize;
+            Scroll.Measure(availableSize);
+            var size = List.DesiredSize;
```

Comment:
I'll probabyl be undoing the GUI scrollbar changes for now, but still want to know: why do ScrollContainers not return desired-size information? Is there any way to easily know how much vertical  space will be taken up by the scrollbar?

Reply:
@PJB3005